### DISCUSSION TITLE
What resources does prune remove?

### DISCUSSION THREAD

**Discussion by sergioisidoro** [fluxcd/flux2#1793] [Q&A] [answered]
It's unclear by the documentation what resources does `prune` (garbage collection) removes.

https://fluxcd.io/docs/components/kustomize/api/

Does it remove resources managed by other services and CRDs (eg. certificates by cert-manager that are not on the repo, or secrets generated by sealed secrets)?

**Comment by stefanprodan** [comment, marked as answer]
Please see the garbage collector docs: https://fluxcd.io/docs/components/kustomize/kustomization/#garbage-collection
